tests/src/runtimeApi/memory/hipMemset2D.cpp

```diff
@@ -113,8 +114,8 @@ int main(int argc, char *argv[])
     hipCtxCreate(&context, 0, p_gpuDevice);
 
     bool testResult = false;
```

Comment:
testResult should be set to true by default for the '&' to work correctly in the following lines.